src/components/login-modal.jsx

```diff
@@ -42,7 +42,7 @@ class Login extends Component {
       let tld = hostname.split('.').slice(-2).join('.');
       if (tld == 'github.io') return null;
       if (tld == 'netlify.com') return null;
-      if (tld == 'localhost') return null;
+      if (tld == 'localhost' || hostname.match(/^127|^0/)) return null;
```

Comment:
Looks fine, but not ideal. What if the website address starts from `127` or `0`? e.g. 127.com, or even `127.0.0.1.com`

Reply:
Or, I could split the hostname on `.` and check if the last index is a digit: `hostname.split('.').pop().match(/[0-9]/)`

Reply:
Hmm, why not just check if hostname exactly matches `127.0.0.1` or `0.0.0.0`?

Reply:
I thought I'll handle cases of `127.x.x.x`

Reply:
@li-boxuan So should I just check for `127.0.0.1` and `0.0.0.0`?

Reply:
I think so, unless there is any other exception such that you have to use regex to match.